src/apps/intermediate-list/modal/my-payment-overview-modal.tsx

```diff
@@ -32,11 +35,7 @@ const MyPaymentOverviewModal: FC = () => {
         <div className="modal-cta__buttons mt-48">
           <Link
             className="btn-primary btn-filled btn-large arrow__hover--right-small"
-            href={
-              new URL(
-                "https://www.borger.dk/vaelg-kommune?actionPageId=065ca8f9-a1f5-4946-ada7-12e163f568df&selfserviceId=7200a519-38ad-48b9-b19a-6e5783e39999"
-              )
-            }
+            href={new URL(paymentOverviewUrl)}
```

Comment:
If the url comes from useUrls (I think that is the name of the function) you do not need to create your own URL object it is already formatted as an URL object